### PR TITLE
Implement rapid healing

### PR DESCRIPTION
**Pull request by Allofich**
Implemented rapid healing (none, in light, in darkness, always) and cleaned up the earlier health recovery formula. This is checked against classic and should be correct.

### CODE REVIEW

File: Assets/Scripts/Internal/WorldTime.cs [hunk 1]
```diff
@@ -46,6 +46,12 @@ public DaggerfallDateTime Now
             get { return DaggerfallDateTime; }
         }
 
+        public bool isDayTime()
```
Comment (Interkarma) [inline Assets/Scripts/Internal/WorldTime.cs]: WorldTime is just a value container. This doesn't belong here, I'm sorry. :) This functionality already exists in DaggerfallDateTime.IsDay. You can check from anywhere through the singleton like so:
    if (DaggerfallUnity.Instance.WorldTime.DaggerfallDateTime.IsDay) { }

**Comment by Interkarma** [comment]
Awesome, thank you for adding that and fixing up formula. 👍

**Comment by Allofich** [comment]
I didn't notice DaggerfallUnity.Instance.WorldTime.DaggerfallDateTime.IsDay. The PR uses it now.

**Comment by Interkarma** [comment]
Cheers mate

**Pull request merged**